pkg/blackduck/ctl_blackduck.go

```diff
@@ -225,6 +225,33 @@ func (ctl *Ctl) AddSpecFlags(cmd *cobra.Command, master bool) {
 	cmd.Flags().StringVar(&ctl.LicenseKey, "license-key", ctl.LicenseKey, "License Key of Black Duck")
 }
 
+var nSpecFlagCmd cobra.Command // command for getting flags for this spec
+var specFlags *pflag.FlagSet   // set of flags for this spec
+var numSpecFlagsChanged int    // number of flags that were set for the spec
+
+// NSpecFlag returns the number of spec flags that were set
+func (ctl *Ctl) NSpecFlag(flagset *pflag.FlagSet) int {
```

Comment:
This is the same function as other commands.  Suggest moving into a base class that can be included in these separate command classes.  No reason to duplicate code.